src/main/MattermostView.js

```diff
@@ -187,6 +188,19 @@ export class MattermostView extends EventEmitter {
     return WindowManager.getMainWindow.webContents;
   }
 
+  handleDidNavigate = (event, url) => {
+    const isUrlTeamUrl = urlUtils.isTeamUrl(this.server.url, url);
+    if (isUrlTeamUrl) {
+      this.setBounds(getWindowBoundaries(this.window));
```

Comment:
seems like we are always calling setbounds with getWindowBoundaries as a parameter, maybe we should have that call into setbounds and pass only the needed parameters?

I'm 0/5 on this one, just noticed that they are always together

Reply:
I'm fine with keeping them separate and having `setBounds` be more generic. I don't think there's any case currently where we wouldn't want to size the view to the size of the window, but there's no harm in having that option.